README.md

```diff
@@ -95,18 +95,23 @@ pip install .
 To build Modulus docker image:
 
 ```bash
-docker build -t modulus:deploy --target deploy -f Dockerfile .
+docker build -t modulus:deploy \
+    --build-arg TARGETPLATFORM=linux/amd64 --target deploy -f Dockerfile .
 ```
 
 Alternatively, you can run `make container-deploy`
 
 To build CI image:
 
 ```bash
-docker build -t modulus:ci --target ci -f Dockerfile .
+docker build -t modulus:ci \
+    --build-arg TARGETPLATFORM=linux/amd64 --target ci -f Dockerfile .
 ```
 
-Alternatively, you can run `make container-ci`
+Alternatively, you can run `make container-ci`.
+
+Currently only `linux/amd64` and `linux/arm64` platforms are supported. `linux/amd64`
```

Comment:
So for ARM, we just dont add a bunch of dependencies? That seems to be the case with the Dockerfile. Should there be a quick comment that some utils wont work in the ARM container?

Reply:
No, all dependencies are supported with our build in ARM too. This Dockerfile will build the dependencies from source that don't have an released ARM wheel/package.